pkg/executables/govc.go

```diff
@@ -972,3 +972,95 @@ func getDeployOptions(network string) ([]byte, error) {
 
 	return deployOpts, err
 }
+
+func (g *Govc) CreateUser(ctx context.Context, username string, password string) error {
```

Comment:
We need function comments on all these even if they are simple. `AddGroupUser` for example feels a little odd. I had to look up what `sso.group.update` does which suggests the name isn't clear with its intent.

Reply:
Sorry, I should've been more elaborate. There's a specific format to Go comments too. This ensures tooling such as `go doc` can render code comments and it provides a really handy wiki for understanding the code base over the CLI. Tools like `godoc` (the web server version) also parse and publish a website similar to `pkg.go.dev`. 

https://tip.golang.org/doc/comment

Reply:
Heres an example that I wrote today for reference (this code may or may not get merged, eventually) https://github.com/aws/eks-anywhere/pull/3246/files

Reply:
Ah, thanks! I've updated the docstrings to match the style that I *think* these examples/docs are demonstrating.